backend/test/utils.js

```diff
@@ -351,13 +351,7 @@ async function createTestEncryptedOfferData(
       id: 'stripe',
       label: 'Credit Card'
     },
-    discountObj: discount
-      ? {
-          code: discount.code,
-          discountType: discount.discountType,
-          value: discount.value
-        }
-      : {},
+    discountObj: discount ? discount : {},
```

Comment:
Isn't that going to break the tests that don't pass a discount?

Reply:
No, it shouldn't. It just passes all the fields of discountObj instead of picking only 3.

Reply:
yes but I was wondering in case no discount is passed as an argument to this method. then discountObj would be set to undefined. which may cause problem?
anyway, I assume you verified all the tests ran so I'm likely wrong.